lib/model/message.dart

```diff
@@ -302,4 +305,24 @@ class MessageStoreImpl with MessageStore {
       view.notifyListenersIfMessagePresent(event.messageId);
     }
   }
+
+  void handleSubmessageEvent(SubmessageEvent event) {
+    final poll = messages[event.messageId]?.poll;
+    if (poll == null) return;
```

Comment:
If the message is missing, that's normal. But if the message is present and not a poll, something's wrong. Let's have separate `if … return;` lines for those, with `// TODO(log)` on the latter.